perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_facility_attribute_def_virt_enabledO365MailForwardTest.java

```diff
@@ -0,0 +1,67 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.*;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.when;
+
+public class urn_perun_user_facility_attribute_def_virt_enabledO365MailForwardTest {
+
+	private static urn_perun_user_facility_attribute_def_virt_enabledO365MailForward classInstance;
+	private static PerunSessionImpl session;
+	private static Attribute enabledMailFwdAttr;
+	private static Attribute disabledMailFwdAttr;
+	private static Attribute o365MailAttr;
+	private static User user;
+	private static Facility facility;
+	private static Resource resource;
+	private static final String exampleMail = "example mail";
+
+
+	@Before
+	public void setUp() {
+		classInstance = new urn_perun_user_facility_attribute_def_virt_enabledO365MailForward();
+		session = mock(PerunSessionImpl.class, RETURNS_DEEP_STUBS);
+
+		user = new User();
+		facility = new Facility();
+		resource = new Resource();
+
+		enabledMailFwdAttr = new Attribute();
+		disabledMailFwdAttr = new Attribute();
+		o365MailAttr = new Attribute();
+		enabledMailFwdAttr.setValue(false);
+		disabledMailFwdAttr.setValue(true);
+		o365MailAttr.setValue(exampleMail);
+	}
+
+	@Test
+	public void getAttributeValueWithEnabledFwdTest() throws Exception {
+		System.out.println("urn_perun_user_facility_attribute_def_virt_enabledO365MailForward.GetAttributeValue()");
+
+		when(session.getPerunBl().getAttributesManagerBl().getAttribute(any(PerunSessionImpl.class), any(Facility.class), any(User.class), eq(AttributesManager.NS_USER_FACILITY_ATTR_VIRT + ":disableO365MailForward"))).thenReturn(enabledMailFwdAttr);
+		when(session.getPerunBl().getAttributesManagerBl().getAttribute(any(PerunSessionImpl.class), any(Facility.class), any(User.class), eq(AttributesManager.NS_USER_FACILITY_ATTR_VIRT + ":o365MailForward"))).thenReturn(o365MailAttr);
```

Comment:
move this to the setup method

Reply:
Moved only the second line as the first one differs between the methods.